src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,206 @@
+$base-color: #000066;
+$white-bg: #fff;
+$link-color: #939393;
+
+.cardOuter {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  padding: 20px;
+  margin: 200px 30px;
+  justify-content: center;
+  background-color: $white-bg;
+  color: $base-color;
+  position: relative;
+  border-radius: 15px;
+}
+
+.cardMain {
+  flex-basis: 70%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.cardImage {
+  flex-basis: 30%;
+  padding: 0 2em;
+
+  img {
+    height: 150px;
+    border-radius: 10px;
+  }
+}
+
+.cardInfo {
+  flex-basis: 70%;
+}
+
+.cardContact {
+  flex-basis: 30%;
+  display: flex;
+  flex-direction: column;
+  padding-top: 1em;
+}
+
+.cardLinks {
+  flex-basis: 100%;
+}
+
+.contactInfo {
+  flex-basis: 100%;
+}
+
+.textIcon {
+  padding-left: 20px;
+  position: relative;
+  vertical-align: bottom;
+  
+  span {
+    display: inline-block;
+    width: 20px;
+    left: 0;
+    position: absolute;
+    top: 0;
+    height: 100%;
+    
+    i {
+      font-size: 1.2em;
+      font-weight: 500;
+      left: 0;
+      position: absolute;
+      top: 0;
+      }
+  }
+}
+
+.card-links {
+  text-align: left;
+  margin: 1em 0;
+  i {
+    color: $base-color;
+    font-size: 2em;
+    margin-right: 1em;
+  }
+}
+
+.trackPerson {
+  position: absolute;
+  right: 20px;
+  top: 0;
+  padding: 5px;
+  
+  p {
+    padding-right: 40px;
+    vertical-align: middle;
+   }
+
+  .fa {
+    position: absolute;
+    top: -5px; 
+    color: #000066;
+    font-size: 2.5em;
+    right: 0;
+ }
+}
+
+
+// Dropdown Button
+
+.btn-outer {
+  position: relative;
+}
+
+.custom-dropdown-btn {
+  color: $link-color;
+  font-weight: bold;
+  cursor: pointer;
+  display: inline-block;
+  padding: 10px 0;
+  transition: all 0.4s;
+  width: fit-content;
+
+  &:hover {
+
+    .custom-dropdown-menu {
+      display: block;
+    }
+  }
+}
+
+.custom-dropdown-menu {
+  z-index: 3;
```

Comment:
Here `z index` makes sense and `position:absolute` as well. good job 👍 but please understand that you don't have to use `position:relative` everywhere.